app/src/main/java/edu/agh/susgame/front/ui/Translation.kt

```diff
@@ -20,9 +26,16 @@ object Translation {
                 return "Oczekuje $n ${if (n == 1) "gracz" else "graczy"}"
             }
         }
-
         const val CREATE_GAME = "Stwórz nową grę"
     }
+    object CreateGame{
```

Comment:
Hmm do tej pory dawałem spację między nazwą obiektu, a `{`

Żeby zachować w miarę spójność kodu, proponuję na razie używać domyślnego formatera z IDE. Po napisaniu kodu można kliknąć na cały pakiet (tylko kod, bez testów) i sformatować wszystko:

![image](https://github.com/Nepommuck/SusGameFrontend/assets/100998310/799d8eec-9b23-4ed8-a24f-fb0571d174db)

![image](https://github.com/Nepommuck/SusGameFrontend/assets/100998310/cb78a0c6-722a-4681-8808-fd8d80040895)

Alternatywnie jak zmieniasz tylko 1 plik, to wystarczy `crtl + alt + l`

Reply:
o własnie nie wiedziałem gdzie sforsowac to domyslne formatowanie, jestem za